integration/network/network_test.go

```diff
@@ -103,3 +109,37 @@ func TestNetworkList(t *testing.T) {
 		})
 	}
 }
+
+func TestAPINetworkGetDefaults(t *testing.T) {
+	skip.If(t, runtime.GOOS == "windows")
+
+	ctx := setupTest(t)
+	apiClient := testEnv.APIClient()
+
+	defaults := []string{"bridge", "host", "none"}
+	for _, netName := range defaults {
+		assert.Assert(t, IsNetworkAvailable(ctx, apiClient, netName))
+	}
+}
+
+func TestAPINetworkFilter(t *testing.T) {
+	skip.If(t, runtime.GOOS == "windows")
+
+	ctx := setupTest(t)
+	apiClient := testEnv.APIClient()
+
+	networkName := "bridge"
```

Comment:
This may be fixable for windows if we make the name of the network depend on platform